### ISSUE TITLE
Reviewers should be able to reply to older versions in a channel for an add-on

### ISSUE THREAD

**Issue by abhn** [mozilla/addons#1981]
### Describe the problem and steps to reproduce it:
Similar to mozilla/addons#1980 add-on reviewers should be able to reply to any version of the add-on for a given channel, and not just the latest.

### What happened?
When replying to an add-on in reviewer tools, the version isn't specify-able and the reply gets attached to the latest (approved?) version. 

### What did you expect to happen?
Reviewers can specify which version they'd like to reply to and the reply gets posted to that version of the add-on for the developer to see.

As some context on this, for unlisted add-ons, the latest approved version might not be the version that's being reviewed, and to be able to specify the version when sending a reply can help reduce the need to specify the version manually in the review message itself.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/AMOENG-267)

**Comment by diox** [comment]
Note that developers also need to be able to reply in the same fashion. There is a potential UX issue here, in devhub it could be a little confusing for developers if the replies are attached to an older version on a different page, but we can live with that edge case. They should be able to reply by email normally.

**Comment by diox** [comment]
Note: per request from Operations during our standup when we discussed this feature, reviewers will explicitly need to select a version when replying now (like other actions which let you select a version). Typically it should be the last one.

In addition, they can select multiple versions to reply to, and this will generate a single activity log, but it will generate multiple emails.

**Comment by ioanarusiczki** [comment]
From reviewers pages it should be fine. When developers reply to reviewers versions are flagged for human review and get a due date. 

On developers side:

- when a reviewer is sending a reply on an older version there's no indication of this on versions page. There's a review history pending count which I saw as being present always for the latest version submitted in the channel. 
The same applies when selecting multiple versions and sending a reviewer reply, for example I've selected version 2.0 and 4.0 and sent a reviewer reply, version 4.0 gets the pending count but version 2.0 won't display that. However, emails are sent individually for each version and the reply is available for the older version too when clicking the review history.

![review history](https://github.com/user-attachments/assets/ccb45c2e-25d2-4c85-b969-79bf77267324)

- when sending developer replies from email for older versions there's an email sent to use https://addons-dev.allizom.org/ for leaving a reply. It's working to send a developer reply from email only for the latest version in the channel. 

![cannot reply try dev hub](https://github.com/user-attachments/assets/a4a3b3a4-9a57-4ca2-9032-1a37efdf92ee)
